test/spec/controllers/reviewRegistration.spec.js

```diff
@@ -40,6 +186,50 @@ describe('Controller: ReviewRegistrationCtrl', function () {
     ).toBe(true);
   });
 
+  describe('acceptedPaymentMethods', () => {
+    it('calculates accepted payment methods for the registrant types', () => {
+      expect(scope.acceptedPaymentMethods()).toEqual({
+        acceptCreditCards: true,
+        acceptChecks: true,
+        acceptTransfers: true,
+        acceptScholarships: false,
+        acceptPayOnSite: false,
+      });
+    });
+
+    describe('acceptPayOnSite', () => {
+      it('is true on incomplete registrations', () => {
+        testData.conference.registrantTypes[1].acceptPayOnSite = true;
+        testData.registration.completed = false;
+        initController();
+
+        expect(scope.acceptedPaymentMethods().acceptPayOnSite).toBe(true);
+      });
+
+      it('is false on completed registrations', () => {
+        testData.conference.registrantTypes[1].acceptPayOnSite = true;
+        testData.registration.completed = true;
+        initController();
+
+        expect(scope.acceptedPaymentMethods().acceptPayOnSite).toBe(false);
+      });
+    });
+
+    it('returns false when no payment methods are accepted', () => {
```

Comment:
Is this the test when the form requires no payment?

Reply:
Yes. Is there anything I can do to make that clearer?

Reply:
No you're good, I think I was just confirming